package.json

```diff
@@ -76,7 +75,7 @@
     "chai-http": "^4.0.0",
     "cheerio": "^1.0.0-rc.3",
     "eslint-config-oclif": "^3.0.0",
-    "http-server": "^0.11.1",
```

Comment:
👋 so long, you won't be missed (thinking back to all of the ecstatic issues on windows with this package)